hackathon_site/hackathon_site/tests.py

```diff
@@ -50,18 +50,26 @@ def _apply_as_user(user, team=None):
 
         application_data = {
             "birthday": date(2000, 1, 1),
-            "gender": "no-answer",
+            "pronouns": "no-answer",
             "ethnicity": "no-answer",
             "phone_number": "1234567890",
+            "city": "Toronto",
+            "country": "Canada",
             "school": "UofT",
             "study_level": "other",
             "graduation_year": 2020,
-            "q1": "hi",
-            "q2": "there",
-            "q3": "foo",
-            "conduct_agree": True,
-            "data_agree": True,
+            "program": "Engineering",
             "resume": "uploads/resumes/my_resume.pdf",
+            "linkedin": "https://www.linkedin.com/feed/",
```

Comment:
minor: you don't need the social media links here because they are optional fields